indy-besu/docs/roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Hyperledger Indy Besu ROADMAP
+
+Note: Right now we have finished PoC implementation. Roadmap tasks and their priorities are still in process of defining.
+
+## Phase 0: PoC
+* Basic transactions (without additional validation) 
+* VDR
+* Migration design 
+* Experiments (eth signature, upgrade, role model, migration)
+
+## Phase 1: MVP
+* Aries Framework Javascript integration Demo
+* Double-check did:indy, did:sov methods compatibility
```

Comment:
Is it really double-check? Or rather support it (on VDR level)?

Reply:
Updated